common-lib/common/signal/README.md

```diff
@@ -36,6 +36,8 @@ Init level:
 
 |Name|Description|Possible values|Example value|Default value|
 |-----|---|---|---|---|
+|datasource| Prometheus Datasource name. |*|`custom_datasource`|`datasource`|
+|datasourceLabel| Prometheus Datasource label. |*|`Custom data source`|`Data source`|
```

Comment:
Should `lokiDatasourceName` and `lokiDatasourceLabel` not also be mentioned here? :)

Reply:
Loki can be renamed in variables, a more generic function. And this is a readme for signals, where only prometheus could be renamed.